kairon/shared/channels/mail/data_objects.py

```
@@ -0,0 +1,125 @@
+            data.pop('user')
+            return data
+        except Exception as e:
+            raise AppException(str(e))
+
+
+    @staticmethod
+    def delete_doc(bot: str, intent: str):
+        try:
+            MailClassificationConfig.objects(bot=bot, intent=intent).delete()
+        except Exception as e:
+            raise AppException(str(e))
```

Ruff (0.7.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)